R/build-home.R

```diff
@@ -79,7 +79,13 @@
 #' @section Badges:
 #' Status badges are displayed in the sidebar under the section "Dev status".
 #' This section is automatically populated if the first paragraph of the
-#' homepage consists solely of status badges as linked images.
+#' homepage source (index.Rmd, index.md, README.Rmd, README.md)
+#' consists solely of status badges as linked images, or if one
+#' paragraph, not necessarily the first, starts with `<!-- badges: start -->`
```

Comment:
I think you could indicate the order of options a little more clearly. Something like: To find badges, pkgdown first looks for a paragraph starting with `<!-- badges: start -->` and ending with `<!-- badges: end -->`, as created by .... Failing that, it looks at the first paragraph, and uses it if it only contains images.